suite-common/wallet-graph/src/graphUtils.ts

```diff
@@ -967,19 +973,18 @@ export const getTimeFrameIntervalsWithSummaryBalances = (
     return graphPoints;
 };
 
-/**
- * react-native-graph library has problems with rendering path when there are some invalid values.
- * Also animated=true graph does not show when dates do not follow each other from the unix epoch
- * (start on 00:00:00 UTC on 1 January 1970).
- *
- */
-export const getValidGraphPoints = (graphPoints: LineGraphPoint[]) =>
-    graphPoints
-        .filter(point => !new BigNumber(point.value ?? '').isNaN())
-        .map((point, index) => ({
-            ...point,
-            date: new Date(index),
-        }));
+// react-native-graph library has problems with rendering path when there are some invalid values.
+export const getValidGraphPoints = (graphPoints: LineGraphPoint[]): LineGraphPoint[] =>
+    graphPoints.filter(point => !new BigNumber(point.value ?? '').isNaN());
```

Comment:
Nit: Wouldn't it be beneficial to use TS-belt functions to process this data based on your recommendations?

Reply:
Probably but it happens only once when legend changes and it's only 80 items, so I think it will be hardly measurable. But I think I will change it anyway.

Reply:
I will keep it this way, there is `readonly` TS error again, which is easily solvable but probably not worth changing it then.